### PR TITLE
Sarah & Elins project-music

### PR DESCRIPTION
**Pull request by Kneedler**
Link to my deployed project: 
https://inquisitive-fudge-c150a0.netlify.app/

My reflections on how this week's project turned out: 
- We are happy with how it turned out, it follows the guidelines. The result is close to the provided design. 

Things I'd like to have clarified or explained in more detail:
- How the props work.

### CODE REVIEW

File: code/src/components/MusicCard.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </div>
+      <h2 className="album-title">
+        <a className="spotify-link album-link" href={props.recordData.external_urls.spotify}>
+          {props.recordData.name}
+        </a>
+      </h2>
+      <h3 className="artist-name">
+        <ArtistName artists={props.recordData.artists} />
+      </h3>
+    </section>
```
Comment (Petrasoderstrom1612) [inline code/src/components/MusicCard.js:37]: Impressive to put all code in one card. It saves space for unneccesary repetition of code.

File: code/src/index.css [hunk 2]
```diff
@@ -1,13 +1,167 @@
...
+  color:#ffffff
+}
+
+/* tablet */
+@media (min-width: 668px)  {
+  .album-wrapper {
+    justify-self: center;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: 1fr 1fr;
```
Comment (Petrasoderstrom1612) [inline code/src/index.css:115]: Very smooth solution

**Review by Petrasoderstrom1612 — COMMENTED** [review]
The code was really good and the design is really close to the expected result based on the criteria!

File: code/public/index.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <title>Project Music Releases</title>
```
Comment (juliaostedt) [inline code/public/index.html:16]: Good job finding and changing this detail

File: code/src/components/ArtistName.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const ArtistName = (props) => {
+  const artistArray = props.artists;
+  const allArtists = artistArray.map((artist, index) => {
+    // If the artist is in the last place of the array and the array of artists is longer than 1 add
+    // '&' before artistname.
+    if (index === artistArray.length - 1 && artistArray.length !== 1) {
```
Comment (juliaostedt) [inline code/src/components/ArtistName.js:8]: Helpful comments to follow along the code. I would love to learn more behind this first part to generate the '&' symbol. Good job!

**Review by juliaostedt — COMMENTED** [review]
Great job this week! Easy to follow your code since you did not have too many components. 👍

**Pull request closed without merging**